CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 2.0.3 (Next)
 
+* [#46](https://github.com/acm19/aws-request-signing-apache-interceptor/issues/46): Add editorconfig - [@acm19](https://github.com/acm19).
```

Comment:
I saw you opened an issue to put here: I usually link to the PR rather than the issue, but also the issue is ok. The only goal of including a link is that a user can click on something and read more about it. They can easily go between issue and PR.